index.html

```diff
@@ -69,12 +72,236 @@ <h1 class="col col-11 col-lg-5 aligne-self-center"> glade to see you</h1>
+                        <img src="./assets/Counter.png" class="dot" alt="dot">
+                        <span class="att2">2015</span>
+                    </div>
+                    <span class="card-text">A daily selection of privately 
+                        personalized reads; no accounts or sign-ups required.</span>
     
+                    <div class="tag-section">
+                        <div class="tag-div">
+                            <span class="tag">html</span>
+                        </div>
+                        <div class="tag-div">
+                            <span class="tag">css</span>
+                        </div>
+                        <div class="tag-div">
+                            <span class="tag">javaScript</span>
+                        </div>
+                    </div>
+                    <a href="#" class="button">See Project</a>
+            
+            </div>
+            
+            
+        </div>
+        <div class="grid-card">
+            
+            <img class="card-img" src="./assets/Snapshoot4.png" alt="Card image">
+            <div class="card-body">
+                <h1 class="card-title">Multi-Post Sorties</h1>
+                    <div class="card-attribut">
+                        <span class="att1">CANOPY</span>
+                        <img src="./assets/Counter.png" class="dot" alt="dot">
+                        <span class="att2">Back End Dev</span>
+                        <img src="./assets/Counter.png" class="dot" alt="dot">
+                        <span class="att2">2015</span>
+                    </div>
+                    <span class="card-text">A daily selection of privately personalized 
+                        reads; no accounts or sign-ups required.</span>
     
-    
-   
+                    <div class="tag-section">
+                        <div class="tag-div">
+                            <span class="tag">html</span>
+                        </div>
+                        <div class="tag-div">
+                            <span class="tag">css</span>
+                        </div>
+                        <div class="tag-div">
+                            <span class="tag">javaScript</span>
+                        </div>
+                    </div>
+                    <a href="#" class="button">See Project</a>
+            </div>
+            
+            
+        </div>
+    </section>
+
+    <div class="about-me">
+        <h1 class="card-title">About</h1>
+        <h1 class="card-title">Myself</h1>
+
+        <span class="card-text">
+            Hello I’m a software developer! 
+            I can help you build a product , 
+            feature or website Look through 
+            some of my work and experience! 
+            If you like what you see and have a 
+            project you need coded, don’t hestiate to contact me.
+        </span>
+        <span class="let">LET'S CONNECT</span> <br>
```

Comment:
- [ ] Titles like the text `LET'S CONNECT` should use a `hx` tag. With the `hx` tag, there won't be need for the `br` tag because the `hx` tags are block level elements while `span` is an inline element

Reply:
- [ ] it is nice you used a hx tag but as mentioned with the reviewer remove br tag is